File: Bua.CodeRev.UserService/Bua.CodeRev.UserService.Core/Models/Contest/InterviewSolutionInfo.cs
using System;
using Bua.CodeRev.UserService.DAL.Entities;

namespace Bua.CodeRev.UserService.Core.Models.Contest
{
    public class InterviewSolutionInfo
    {
        public Guid Id { get; set; }
        public string Vacancy { get; set; }
        public string InterviewText { get; set; }
        public long InterviewDurationMs { get; set; }
        public bool IsStarted { get; set; }
    }
}

File: Bua.CodeRev.WebClient/src/app/contest/models/interviewSolutionInfo.ts
import { InterviewSolutionInfoResposne } from './response/interviewSolutionInfo-response';

export class InterviewSolutionInfo {
    public id: string;
    public vacancy: string;
    public interviewText: string;
    public interviewDurationMs: number;
    public isStarted: boolean;
    public startTimeMs: number;
    public endTimeMs: number;

    constructor(resp: InterviewSolutionInfoResposne) {
        this.id = resp.id ?? '';
        this.vacancy = resp.vacancy ?? 'none';
        this.interviewText = resp.interviewText ?? '';
        this.interviewDurationMs = resp.interviewDurationMs ?? -1;
        this.isStarted = resp.isStarted ?? false;
        this.startTimeMs = resp.startTimeMs ?? -1;
        this.endTimeMs = resp.endTimeMs ?? -1;
    }
}